stl/inc/format

```diff
@@ -1620,11 +1663,26 @@ public:
             _Throw_format_error("Invalid type specification.");
         }
         const char _Narrow_type = static_cast<char>(_Type);
-        enum class _Presentation_type_category { _Default, _Integer, _Floating, _String, _Pointer, _Char };
+        enum class _Presentation_type_category {
+            _Default,
+            _Integer,
+            _Floating,
+            _String,
+            _Pointer,
+            _Char,
+#if _HAS_CXX23
+            _Escape,
+#endif // _HAS_CXX23
+        };
         auto _Cat = _Presentation_type_category::_Default;
         switch (_Narrow_type) {
         case '\0':
             break;
+#if _HAS_CXX23
+        case '?':
+            _Cat = _Presentation_type_category::_Escape;
+            break;
+#endif // _HAS_CXX23

```

Comment:
Potential future improvement:

We could recognize this in c++20 and earlier modes and emit a helpful diagnostic.